core/utils/poly-look/src/visualisations/charts/treeMap/treeMap.js

```diff
@@ -91,9 +91,11 @@ export class TreeMap extends Chart {
       .text((d) => `${d.data.name}:\n${d.data.value}`)
       .attr("font-size", this._fontSize)
       .attr("y", this._fontSize)
+      .attr("x", defaultPadding)
       .attr("fill", this._fontColor);
 
     const onUnfittingText = this._onUnfittingText;
+    texts.call(wrapTexts);
```

Comment:
Is there any functional difference in moving it here? If there's not, maybe it can be moved one line up?